main.cpp

```diff
@@ -133,15 +133,15 @@ bool printImage(char option, int width = 5)
 /*
  5)
  */
-char classRoomStudent( int numberID, int year = 2020)
+char getClassRoomStudent( int numberID, int year = 2020)
 {
     ignoreUnused(numberID, year);
     return {};  
 }
 /*
  6)
  */
-bool shiningStar(int numberID, int temperature = 1000)
+bool starEmitsLight(int numberID, int temperature = 1000)
```

Comment:
This function name is describing the state of something. 
choose a different action and convert it into code
example: 
Compute the amount of light emitted by a star. 
```
computeAmountOfLightEmittedByStar()
```
- [ ] revise